DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/SegmentCollectionToSpeckleConverter.cs

```diff
@@ -34,27 +28,39 @@ public SOG.Polyline RawConvert(ACG.ReadOnlySegmentCollection target)
     {
       len += segment.Length;
 
-      // do something specific per segment type
+      // specific conversion per segment type
       switch (segment.SegmentType)
       {
         case ACG.SegmentType.Line:
-          points.Add(_pointConverter.RawConvert(segment.StartPoint));
-          points.Add(_pointConverter.RawConvert(segment.EndPoint));
-          break;
-        case ACG.SegmentType.Bezier:
-          var segmentBezier = (ACG.CubicBezierSegment)segment;
-          points.AddRange(_bezierConverter.RawConvert(segmentBezier).GetPoints());
-          break;
-        case ACG.SegmentType.EllipticArc:
-          var segmentElliptic = (ACG.EllipticArcSegment)segment;
-          points.AddRange(_arcConverter.RawConvert(segmentElliptic).GetPoints());
+          points = AddPtsToPolylinePts(
+            points,
+            new List<SOG.Point>()
```

Comment:
`AddPtsToPolylinePts` doesn't need to return the list. List are passed by reference, so the list that the method is modifying is the same as the one in the `points` variable`